extensions/sql-database-projects/package.vscode.json

```diff
@@ -3,5 +3,29 @@
 	"extensionDependencies": [
 		"ms-mssql.mssql",
 		"Microsoft.data-workspace-vscode"
+	],
+	"contributes": {
+		"commands": [
```

Comment:
@Charles-Gagnon my Product validation build failed I think it's related to the change in this file. I'm not sure what I did wrong. do you have any idea? this is the error:
[19:14:37] TypeError in plugin "gulp-json-editor"
Message:
    Cannot convert undefined or null to object
Details:
    domainEmitter: [object Object]
    domainThrown: false

error Command failed with exit code 1.

https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=114772&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=d2c76452-9b5b-53aa-b9e7-4eac61843d9f

Reply:
Not offhand. You can run the commands yourself though to debug 
Run these once to ensure all the necessary files are there
```
      yarn gulp package-rebuild-extensions
      yarn gulp compile-extensions
```

Then run `yarn gulp package-external-extensions` as needed to debug.

Although a couple things : 

1. You have `view/item/context` as a top-level object. But it should be under the `menus` contribution
2. I do want to point out that you appear to be wanting to add new contributions to the existing list (i.e. you're adding a new command but not wanting to delete all the old ones). 

As https://github.com/microsoft/azuredatastudio/blob/main/build/gulpfile.sql.js#L122 mentions the code does not currently support doing this. It only supports directly overwriting an entire top-level object. So you'll need to make additional modifications if this is a scenario you need.

But generally for commands and menus I would instead suggest just adding it to the root package.json and using the azdataAvailable context key in the when clause to control when these items are shown. Although then I'd have to ask - why would this functionality not be available in ADS as well? Seems like anything in VS Code should also be available in ADS.